lib/ansible/modules/database/postgresql/postgresql_privs.py

```diff
@@ -541,7 +557,9 @@ def main():
             port=dict(type='int', default=5432),
             unix_socket=dict(default='', aliases=['login_unix_socket']),
             login=dict(default='postgres', aliases=['login_user']),
-            password=dict(default='', aliases=['login_password'], no_log=True)
+            password=dict(default='', aliases=['login_password'], no_log=True),
+            ssl_mode=dict(default='disable'),
+            ssl_rootcert=dict(default='')
```

Comment:
default here is empty string but both in the documentation and in the other file, postgresql_user.py , it's None.  Should standardize on either None or empty string in all places.